README.md

```diff
@@ -1,1329 +1,100 @@
-  * [Phosphor Signaling 101](#phosphor-signaling-101)
-  * [A simple react button](#a-simple-react-button)
-  * [subscribing to a signal](#subscribing-to-a-signal)
-* [6 Custom Kernel Interactions: Kernel Managment and Messaging](#6-custom-kernel-interactions-kernel-managment-and-messaging)
-  * [Component Overview](#component-overview)
-  * [Initializing and managing a kernel session (panel.ts)](#initializing-and-managing-a-kernel-session-panelts)
-  * [Executing code and retrieving messages from a kernel (model.ts)](#executing-code-and-retrieving-messages-from-a-kernel-modelts)
-  * [Connecting a View to the Kernel](#connecting-a-view-to-the-kernel)
-  * [How does it look like](#how-does-it-look-like)
-
-## Introduction
-
-This is a short tutorial series about JupyterLab extensions. Jupyterlab can be
-used as a platform to combine existing data-science components into a new
-powerful application that can be deployed remotely for many users. Some of the
-higher level components that can be used are text editors, terminals,
-notebooks, interactive widgets, filebrowser, renderers for different file
-formats that provide access to an enormous ecosystem of libraries from
-different languages.
-
-## Prerequesites ##
-
-Writing an extension is not particularly difficult but requires very basic
-knowledge of javascript and typescript and potentially Python.
-
-_Don't be scared of typescript, I never coded in typescript before I touched 
-JupyterLab but found it easier to understand than pure javascript if you have a 
-basic understanding of object oriented programming and types._
-
-## 1 Hello World: Setting up the development environment ##
-
-#### The template folder structure ####
-
-Writing a JupyterLab extension usually starts from a configurable template. It
-can be downloded with the `cookiecutter` tool and the following command:
-
-```bash
-cookiecutter https://github.com/JupyterLab/extension-cookiecutter-ts
-```
-
-`cookiecutter` asks for some basic information that could for example be setup
-like this:
-```bash
-author_name []: tuto
-extension_name [JupyterLab_myextension]: 1_hello_world
-project_short_description [A JupyterLab extension.]: minimal lab example
-repository [https://github.com/my_name/JupyterLab_myextension]: 
-```
-
-The cookiecutter creates the directory `1_hello_world` [or your extension name]
-that looks like this:
+The goal of this repository is to show how to develop extensions on [JupyterLab](https://github.com/jupyterlab/jupyterlab). 
+It is presented as short tutorial series.
 
 ```bash
-1_hello_world/
-
-├── README.md
-├── package.json
-├── tsconfig.json
-├── src
-│   └── index.ts
-└── style
-    └── index.css
+git clone https://github.com/jtpio/jupyterlab-extension-tutorial.git && 
+  cd jupyterlab-extension-tutorial && \
+  conda env create -f environment.yml && \
+  conda activate jupyterlab-extension-tutorial && \
+  cd hello_world && \
+  npm install && \
+  npm run build && \
```

Comment:
Maybe we can use `jlpm` instead of `npm`, so it's the same as in the [extension cookiecutter](https://github.com/jupyterlab/extension-cookiecutter-ts/tree/master/%7B%7Bcookiecutter.extension_name%7D%7D#install-1)?